### PR TITLE
Use babel preset in cli

### PR DESCRIPTION
**Pull request by doug-wade**
Transpile the cli (but not its clients) with babel-preset-react-server

### CODE REVIEW

**Comment by doug-wade** [comment]
This is blocking on 0.4.0 to publish

**Comment by gigabo** [comment]
@doug-wade - Blocking on 0.4.0 of what?

**Comment by doug-wade** [comment]
@gigabo React Server!

**Comment by gigabo** [comment]
It requires breaking changes?

**Comment by doug-wade** [comment]
0.3.3, 0.4.0, or 1.0.0, whichever comes first

File: package.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
     "jasmine": "2.3.1",
     "jsdom": "5.6.1",
     "jsx-loader": "0.12.2",
-    "lerna": "2.0.0-beta.23",
+    "lerna": "git://github.com/lerna/lerna.git#master",
```
Comment (doug-wade) [inline package.json]: build is passing locally with my lerna built off master (which runs `prepublish` as part of bootstrap), but not on ci so I tried pulling from master.  However, lerna's master branch isn't a functioning node module, so this isn't working either.  Next, I'll try a `lerna run prepublish` on ci, but if that doesn't work, we may just need to wait for lerna-2.0.0-beta.24

**Pull request merged**